### ISSUE TITLE
Does not display on the ship or within the company

### ISSUE THREAD

**Issue by Paradox75831004** [riceisacereal/MinimumQuotaFinder#1]
We have a large collection of mods, there are various loot. But for some reason he does not allocate it and does not show how much is needed for an optimal quota

Our build code is 018d581b-7659-aad2-a990-dccddf2b0389

![image](https://github.com/riceisacereal/MinimumQuotaFinder/assets/72872478/8fbdf0f8-14e3-4520-a516-58e5afdd75a7)

**Comment by NilaierMusic** [comment]
Can confirm this bug even with unmodded loot with only one mod (and dependency). Both the mod and its dependency load into the game without any problems, but it seems like there might be some sort of a logical error or oversight in the code that prevents the core mechanics from working as intended. No errors or warnings in the console (aside from vanilla ones).
![Lethal_Company_iQDXvN0Nxb](https://github.com/riceisacereal/MinimumQuotaFinder/assets/86863646/e28454ed-32f3-4842-b5b9-f8e5cfbf27bf)
![Lethal_Company_bVq98veUdA](https://github.com/riceisacereal/MinimumQuotaFinder/assets/86863646/a443c79c-d62b-4c58-9cce-c86440c84d29)
![Lethal_Company_uWUxOIfKrQ](https://github.com/riceisacereal/MinimumQuotaFinder/assets/86863646/e6396a67-8042-479d-9651-be3b4b29de51)

**Comment by riceisacereal** [comment]
The mod only triggers when you press H. We realized that we didn't make that too clear :P Does it work if you press H?

**Comment by Paradox75831004** [comment]
I have this button already in Windows, can't I do a right-click scan like in that fashion?

**Comment by Paradox75831004** [comment]
I was inconsiderate, I'm sorry. But each time you press a separate key, it is not very convenient and intuitive for a beginner who just shakes the mod - seeing the description is cool, without hints

**Comment by riceisacereal** [comment]
We've decided to separate it from the normal scanning function because the calculations can take quite a while when you have a lot of scrap. We've currently emphasized the fact that you have to press H to scan it more in our README (not on Thunderstore yet). Further, we are thinking about adding a tooltip in the top right of the screen along with the other ones (like press G to drop item), and maybe displaying a message at the start of the game as well.

**Comment by Paradox75831004** [comment]
When I take the items marked with the H button from the ship to the company's cash register (but do not put them down). After switching off and on, it shows other things that are on the ship, and not outside the company. Can this be fixed? So that they take into account all things, not only on the ship but also within the company

**Comment by riceisacereal** [comment]
Would it be possible to provide a recording of what is happening? We do consider scrap within the ship and outside the ship when on the company moon. It should also take into account the value of what is already sold.

**Comment by Paradox75831004** [comment]
You're right about the instructions. Can you make an alert message on the company's planet? ""Press H to mark items for optimal quota" or something similar

**Comment by Paradox75831004** [comment]
When you take out the marked loot near the cash register, but do not sell it. If you turn off the loot display and turn it on again, it highlights not the loot that is already at the checkout, but the one that is on the ship. You don't even need to show the recording here. I hope it will be clear in words

**Comment by Paradox75831004** [comment]
The mod is great, I'll replace it with the one that I scanned with the right mouse button. Not the most convenient.

**Comment by Luesewr** [comment]
https://github.com/riceisacereal/MinimumQuotaFinder/assets/78808564/c45605a5-ff39-4e6f-8be1-3d5d287c28a5

I'm trying to replicate what you're describing but I'm not having any issues

**Comment by Paradox75831004** [comment]
You need to try it on my build, maybe something conflicts with something. Either everything is fine
